src/Components/HistoricalDataChart/HistoricalDataChart.js

```diff
@@ -76,43 +76,47 @@ export const HistoricalDataChart = ({ inventoryId }) => {
     };
 
     const displayError = () => {
-        return <EmptyState className='loadingErrorContainer' variant={EmptyStateVariant.small}>
-            <Title headingLevel="h2" size="lg">
-                Something went wrong
-            </Title>
-            <EmptyStateBody>
-                There was a problem while requesting historical data. Please try again later.
-            </EmptyStateBody>
-        </EmptyState>;
+        return  <FlexItem  alignSelf={{ default: 'alignSelfBaseline' }}>
+            <EmptyState className='loadingErrorContainer' variant={EmptyStateVariant.small}>
+                <Title headingLevel="h2" size="lg">
+                    Something went wrong
+                </Title>
+                <EmptyStateBody>
+                    There was a problem while requesting historical data. Please try again later.
+                </EmptyStateBody>
+            </EmptyState>
+        </FlexItem>;
     };
 
     const displayChart = () => {
-
         return  chartData.length === 0 ?
-            <Bullseye className='loadingErrorContainer'>
-                <Spinner size="xl" aria-labelledby="loading-historical-chart"/>
-            </Bullseye> :
-            <Fragment>
-                <span className='dropdownContainer'>
-                    <Tooltip content={<div>Scroll and pan to zoom and move</div>}>
-                        <OutlinedQuestionCircleIcon size='sm' />
-                    </Tooltip>
-                    <Dropdown
-                        className='dateDropdown'
-                        toggle={
-                            <DropdownToggle
-                                id='chart-date-toggle'
-                                onToggle={onToggle}
-                                toggleIndicator={CaretDownIcon} >
-                                {`Last ${dateRange === DATE_RANGE_7_DAYS ? DATE_RANGE_7_DAYS : RANGE_DROPDOWN_45_DAYS} Days`}
-                            </DropdownToggle>
-                        }
-                        isOpen={isOpen}
-                        dropdownItems={dropdownItems}
-                    />
-                </span>
-
-                <div style={{ height: '275px' }}>
+            <FlexItem  alignSelf={{ default: 'alignSelfBaseline' }}>
+                <Bullseye className='loadingErrorContainer'>
+                    <Spinner size="xl" aria-labelledby="loading-historical-chart"/>
+                </Bullseye>
+            </FlexItem> :
+            <Flex direction={{ default: 'column' }} alignSelf={{ default: 'alignSelfBaseline' }} align={{ default: 'alignRight' }}>
+                <FlexItem align={{ default: 'alignRight' }}>
+                    <span className='dropdownContainer'>
```

Comment:
@kgaikwad can you please remove the classname from here as we have already removed the classes?

Reply:
@PreetiW  - yeah, make sense! I can remove these classes if those classes are not used anywhere else in JS code.